### DISCUSSION TITLE
Chinese Simplified

### DISCUSSION THREAD

**Discussion by Ipmlosion** [DexterHuang/CyberCodeOnline#2640] [General]
没有人做简中吗？
没有人的话我打算用繁中改一版出来
以及README.md能进行本地化吗？
能的话我一起做了

**Comment by DexterHuang** [comment]
可以啊 翻譯都沒有特定人的 隨時任何人可以參與👍

Readme 可以的翻譯嗎？@@ 我不知道github有沒有這個功能

**Reply by Ipmlosion** [reply]
严格来说不是翻译，是可以跳转的简中版

像是：English｜简体中文
（简中显示为蓝色跳转）

用英中举例子，实际上完全可以添加更多语种

**Comment by Ipmlosion** [comment]
@DexterHuang 以及是否对CyberCode Online这个名称进行汉化

**Reply by DexterHuang** [reply]
> @DexterHuang 以及是否对CyberCode Online这个名称进行汉化

在台灣Google Play叫做 賽博原碼

**Reply by Ipmlosion** [reply]
@DexterHuang ？我👀《Cyberpunk 2077》在台湾翻成《電馭叛客 2077》还以为你们不叫赛博朋克呢💦

**Comment by DexterHuang** [comment]
cyberpunk應該就是賽博龐克 但是台灣廠商取電影名字或是遊戲名字一項會比較會取他們覺得比較好的不會照英文直翻
電馭叛客 2077 是遊戲名的意思@@

**Comment by Ipmlosion** [comment]
@DexterHuang 
![Screenshot_20220619_002840_com.foxdebug.acode_edit_1371748837394852.jpg](https://user-images.githubusercontent.com/75523567/174448973-b75cee6c-925c-45f5-a575-a988fc0000da.jpg)
我人已经看傻了

![Screenshot_20220619_003618_com.foxdebug.acode_edit_1371761458055788.jpg](https://user-images.githubusercontent.com/75523567/174448798-d8e47aee-2478-4b16-9e10-0f9e8b7c3baf.jpg)
这甚至有两个简中？

咱规范下怎么样（
不需要的东西删减掉，比如那一大串意义不明的分支（
以后维护也会轻松很多

**Reply by DexterHuang** [reply]
分支應該不會有什麼影響 大部分幫忙改東西的人都不是黑會用
有兩個可能是有人弄到我也沒發現吧

**Reply by zsrcici** [reply]
@DexterHuang 唯一的影响是简中没有怪物和装备的名字 我看见zh-cn文件夹下有一个zh-cn文件夹，我不会弄，可以帮忙移动一下相应的文件吗

**Reply by Ipmlosion** [reply]
> @DexterHuang 唯一的影响是简中没有怪物和装备的名字 我看见zh-cn文件夹下有一个zh-cn文件夹，我不会弄，可以帮忙移动一下相应的文件吗

这个不急，我打算完全重构一份出来（
现在这版lang问题太多了

**Comment by Ipmlosion** [comment]
@DexterHuang 是的，暂时并无太大影响，但我们依旧需要对它进行规范，这也是对以后的维护与社区负责。
建议写份CONTRIBUTING，README 文件帮助人们来认识项目，而 CONTRIBUTING 文件则是告诉人们对项目如何做贡献。它解释了目前项目需要什么样类型的贡献者，社区的流程是什么样的。
可以从创建GitHub账户开始到提交Pull Request，以避免出现现在这种情况（

**Comment by zsrcici** [comment]
我英文不好但我可以帮忙简体化或内陆本地化，如果有需要请在游戏里私聊我：zsrcici

**Comment by SkyNetX007** [comment]
刚入坑，游戏没玩多少，光顾着改翻译了😂
这两天对CCO的简中翻译做了不少删改，现在已经处于能用状态了
大佬重构的lang有眉目了嘛？我也想发PR。如果在重构的基础上维护就没有历史包袱了 @Ipmlosion
现在主要缺的是lore的内容翻译，一个人翻译比较折磨，而且繁中有不少是原创的而非基于英文版，这部分lore计划是繁中特供还是以后会同步到英文和其他语种？@DexterHuang

**Comment by DexterHuang** [comment]
@SkyNetX007 lore基本上沒有要求同步 因為每個地區會有自己的梗之類的有的很難翻譯 還有比如說你只會中文也可以直接寫中文版不需要在乎英文有沒有
